fs-sim/src/System/FS/Sim/FsTree.hs

```diff
@@ -11,7 +12,7 @@
 --
 -- > import Test.Util.FS.Sim.FsTree (FsTree)
```

Comment:
We need to change the `import` above.

Reply:
Changed (and in other `System.FS.Sim.*` modules as well